recipes/flecs/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
```

Comment:
```suggestion
        cmake = CMake(self)
        cmake.definitions["FLECS_SHARED"] = self.options["flecs"].shared
```

Forgot that it needs also this in test_package https://github.com/SpaceIm/conan-flecs/blob/f633f0c705141e8eea726db9ca713bc780108f95/test_package/conanfile.py#L11